composer.json

```diff
@@ -17,15 +17,15 @@
     }
   ],
   "require": {
-    "php": "^7.3",
+    "php": "^7",
     "doctrine/migrations": "~2.2",
-    "illuminate/config": "^8.0",
-    "illuminate/contracts": "^8.0",
-    "illuminate/console": "^8.0",
-    "laravel-doctrine/orm": "^1.7"
+    "illuminate/config": "^6|^7|^8" ,
+    "illuminate/contracts": "^6|^7|^8",
+    "illuminate/console": "^6|^7|^8",
+    "laravel-doctrine/orm": "^1"
```

Comment:
minor version spec should be include still (7.2.5 matches L6 & L7)
```
    "php": "^7.2.5",
    "doctrine/migrations": "~2.2",
    "illuminate/config": "^6.0|^7.0|^8.0" ,
    "illuminate/contracts": "^6.0|^7.0|^8.0",
    "illuminate/console": "^6.0|^7.0|^8.0",
    "laravel-doctrine/orm": "^1.5"
 ```
 
 and then for php 8 testing we just tag on `|8.0`
```
    "php": "^7.2.5|8.0",
    "doctrine/migrations": "~2.2",
    "illuminate/config": "^6.0|^7.0|^8.0" ,
    "illuminate/contracts": "^6.0|^7.0|^8.0",
    "illuminate/console": "^6.0|^7.0|^8.0",
    "laravel-doctrine/orm": "^1.5"
 ```

Reply:
Better tag ^8.0, you don't want to support only 8.0.

When you will add ^8.0? Can't test it at this moment.

Reply:
@Jurigag released now